src/test/java/Homework20.java

```diff
@@ -0,0 +1,80 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.time.Duration;
+
+public class Homework20 extends BaseTest{
+    @Test
+    //Homework # 19 change of the code with "waits-"
+    public void deletePlaylist() throws InterruptedException {
```

Comment:
Please, delete `throws InterruptedException`

**Thread. sleep is bad! It blocks the current thread and renders it unusable for further work.
Using Thread. sleep() frequently in an automation framework is not a good practice.**

Please use the Explicit Wait command, the WebDriver is directed to wait until a certain condition occurs before proceeding with executing the code.

![image](https://user-images.githubusercontent.com/104761482/229338208-fedc3ae9-5c8d-47c2-86fa-78070301f865.png)

![image](https://user-images.githubusercontent.com/104761482/229338228-9d212926-9cd4-4723-868b-a2bfb1910ec4.png)

Reply:
Hi, That was my bad, I missed it. Thank you, i changed and added commits. Thank you!

Reply:
The code is not compiled! Please finish the homework!You got this problem, please correct it. 

![image](https://user-images.githubusercontent.com/104761482/229367396-e8202893-e220-4f8f-b1f4-4a9ffa40c1c8.png)